pyproject.toml

```diff
@@ -0,0 +1,107 @@
+[tool.poetry]
+name = "aio-pika"
+version = "9.0.0"
+description = "Wrapper for the aiormq for asyncio and humans"
```

Comment:
Wrapper **over** aiormq ... would be better, I guess.

Reply:
around?

Reply:
Yes, it's suitable 👍🏻